src/app/api/utils/auth.tsx

```diff
@@ -193,8 +196,24 @@ export const authOptions: AuthOptions = {
             },
           });
 
+          const enabledFlags = await getEnabledFeatureFlags({
+            email: verifiedSubscriber.primary_email,
+          });
+
           await initEmail(process.env.SMTP_URL);
-          await sendEmail(data.recipientEmail, subject, emailTemplate);
+          await sendEmail(
+            data.recipientEmail,
+            subject,
+            enabledFlags.includes("RedesignedEmails")
+              ? renderEmail(
+                  <SignupReportEmail
+                    l10n={l10n}
+                    breachedEmailAddress={profile.email}
+                    breaches={unsafeBreachesForEmail}
+                  />,
+                )
+              : emailTemplate,
+          );
```

Comment:
I like this approach, do you think that this flag should be checked in the `/admin/email` test tool as well? Or maybe that tool should allow choosing old/new email.

Reply:
It would be neat, but the admin tool doesn't support rendering the old templates yet. And since QA has apparently already tested most of the other emails, I think we're actually pretty close to phasing out the old emails, so it's probably not worth effort of adding that :)